### PR TITLE
Update changed link

### PR DESCRIPTION
**Pull request by robtuley**
This PR simply updates a link that has changed. Apologies I should have left a redirect there. 

EDIT: the PR link checker status check was failing, so this PR also adds a fix. This looks like it was introduced in #471 but looks like it has a bug in it when multiple *.md files are touched: the `find` command returns a newline separated list of md files when what is required is a space spearated one so just added a quick fix for that.

### CODE REVIEW

File: patterns/2-structured/service-vs-library.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ reluctant to join forces even if there is significant overlap in requirements.
...
+- They are organised in fully functional DevOps teams: Each team is responsible for their contributions end-to-end, including maintenance, on-call and customer support.
+- A team is tasked with providing a service to their downstream customers that is fairly similar to an existing service built by another team.
 
 ## Forces
 
-* Organisational escalation paths may be different for each of the teams.
-* Members of each team may be unwilling to answer on-call support for errors that do not affect their own downstream customers.
-* Severity levels for the same types of errors may be different across team boundaries due to different SLA definitions per team/customer relationship.
-* Teams may have different security or regulatory constraints governing their deployments.
+- Organisational escalation paths may be different for each of the teams.
```
Comment (spier) [inline patterns/2-structured/service-vs-library.md]: Did you change the bullet format on purpose?

File: translation/ja/patterns/service-vs-library.md [hunk 2]
```diff
@@ -50,25 +50,25 @@ DevOps環境のチームは、サービスのダウンタイムに対応する
...
 
-このパターンに関連するものとして、上記を解決するために異なるアプローチをとる[30日の保証期間](30-day-warranty.md) パターンがあります。
+このパターンに関連するものとして、上記を解決するために異なるアプローチをとる[30 日の保証期間](30-day-warranty.md) パターンがあります。
 
 ## 事例
 
-* ユーロスペース AG
-* Flutter Entertainment: [Flutter インナーソースの応用](https://innersource.flutter.com/start/setup-ci/)には、チーム横断的にコントリビュートする共有コードの"サービスリポジトリ"と、共有リリース成果物をビルドして公開するためのCIパイプラインがあります。各チームは、独自のデプロイメントを定義する"デプロイメント設定"リポジトリを持っています。これは、さまざまな規制要件、サービスおよびインシデント管理の実践、ビジネスの各領域におけるインフラストラクチャのスキルセットによって異なる運用がされています。
+- ユーロスペース AG
+- Flutter Entertainment: [Flutter インナーソースの応用](https://innersource.flutter.com/sdlc/)には、チーム横断的にコントリビュートする共有コードの"サービスリポジトリ"と、共有リリース成果物をビルドして公開するための CI パイプラインがあります。各チームは、独自のデプロイメントを定義する"デプロイメント設定"リポジトリを持っています。これは、さまざまな規制要件、サービスおよびインシデント管理の実践、ビジネスの各領域におけるインフラストラクチャのスキルセットによって異なる運用がされています。
```
Comment (spier) [inline translation/ja/patterns/service-vs-library.md]: Looks like new whitespace is introduced here between Japanese and English characters.

Did you introduce these on purpose, or is this something that maybe your editor did automatically?

**Review by spier — COMMENTED** [review]
Thanks for the link fix @robtuley!

Also great that you found the bug in the GitHub Action. Great catch!
Looks like we need to look for better ways to test those :)

I left two comments inline where it looks like you changed some markdown formatting that isn't necessarily related to this PR. As part of that is related to the Japanese translation, and I don't speak that language, I am not sure if this will look correct to Japanese readers.

File: patterns/2-structured/service-vs-library.md (hunk 1 above)
Comment (robtuley) [reply patterns/2-structured/service-vs-library.md]: IDE markdown auto-formatter artefact, now reverted.

File: translation/ja/patterns/service-vs-library.md (hunk 2 above)
Comment (robtuley) [reply translation/ja/patterns/service-vs-library.md]: yes, IDE auto-formatter. Argh, reverted.

**Comment by robtuley** [comment]
Yes my IDE made a mess of that by having a different markdown auto-formatter applying on save to the files I touched. Reverted those auto-changes now so diff should just be 2 x URL corrections and single-line bugfix to PR url checker.

**Review by spier — APPROVED** [review]

**Comment by spier** [comment]
> Yes my IDE made a mess of that by having a different markdown auto-formatter applying on save to the files I touched. Reverted those auto-changes now so diff should just be 2 x URL corrections and single-line bugfix to PR url checker.

Awesome. Your fix is merged now.

Would there be a way to tell your IDE with markdown formatting we use?
We have this file but apparently more than that is required? https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/.markdownlint.json

**Pull request merged**